lib/octokit/client/repositories.rb

```diff
@@ -26,6 +26,7 @@ def repository?(repo, options = {})
       # @param repo [Integer, String, Hash, Repository] A GitHub repository
       # @return [Sawyer::Resource] Repository information
       def repository(repo, options = {})
+        options = ensure_api_media_type(:template_repositories, options)
```

Comment:
The more I think about it, I'd rather leave out the preview here, if someone wants the new data they can opt into it.

Mind if we remove this line?

Reply:
Do you mean replace it with a check like the others methods?